projects/demo/src/pages/kit/date-range/examples/4-range-separator/mask.ts

```diff
@@ -0,0 +1,8 @@
+import {maskitoDateRangeOptionsGenerator} from '@maskito/kit';
+
+export default maskitoDateRangeOptionsGenerator({
+    mode: 'dd/mm/yyyy',
+    minLength: {day: 3},
+    maxLength: {month: 1},
```

Comment:
I think we can drop `minLength` & `maxLength` from this example.

These props are already well-described in [the previous example](https://tinkoff.github.io/maskito/kit/date-range#min-max-length).
Let's keep the new example as simple as possible :)

Reply:
fixed